development-vm/replication/sync-aws-postgresql.sh

```diff
@@ -0,0 +1,98 @@
+#! /usr/bin/env bash
+#
+# Fetch the most recent PostgreSQL database dump from the given server.
+#
+
+set -eu
+
+USAGE_LINE="$0 [options] SRC_HOSTNAME"
+USAGE_DESCRIPTION="Load the most recent PostgreSQL dump files from the Integration backup bucket."
+. ./common-args.sh
+
+if $SKIP_POSTGRES; then
+  exit
+fi
+
+shift $(($OPTIND-1))
+
+if [ $# -ne 1 ]; then
+  error "A source hostname is required"
+  echo
+  usage
+  exit 2
+fi
+
+SRC_HOSTNAME=$1
+
+POSTGRESQL_DIR="${DIR}/postgresql/"
+
+status "Starting PostgreSQL replication from AWS backups"
+
+if ! $SKIP_DOWNLOAD; then
+  aws s3 sync s3://govuk-integration-database-backups/postgres/$(date '+%Y-%m-%d')/ $POSTGRESQL_DIR/
```

Comment:
I see we used to have a `mkdir -p $POSTGRESQL_DIR` step. Does the aws sync create the directory if it doesn't exist?

Reply:
That is correct